lib/aggregate.js

```diff
@@ -35,6 +35,8 @@ Aggregate.prototype._apply = function(event, isNew) {
 	}
 
 	this._version++;
+    event.version = this._version;
```

Comment:
it should be version on command which is important. Version on event should be from Tapeworm and eventSTream?